apps/docs/content/components/ComparisonTable.mdx

```diff
@@ -124,6 +124,10 @@ For granular control over heading markup or appearance, we recommend using [`Hea
 
 By default, the first data column will be marked as `featured`. This is to draw attention to a particular product as the basis of a comparison against adjacent columns.
 
+Featured columns will have the visually-hidden text `featured` appended to their column title. This text is used to identify the featured column to screen reader users and can be customized using the `visuallyHiddenFeaturedLabel` prop.
+
+```jsx live
```

Comment:
FYI This isn't closed properly so is breaking docs. Guessing it's not needed?

```suggestion
```

![Screenshot 2024-08-14 at 15 21 14](https://github.com/user-attachments/assets/428e1005-4da8-4d4b-a366-0ff414ab43e3)

Reply:
Ah sorry, Copilot has been really eager to add this every opportunity it gets. I keep deleting them but obviously I missed it this time 🤦‍♂️